### PR TITLE
added docs for grant aws API

### PR DESCRIPTION
**Pull request by vikhyat187**
Date: 13th Nov

Developer Name: Vikhyat

---

## Issue Ticket Number
https://github.com/Real-Dev-Squad/todo-action-items/issues/253

## Description
This is the API docs for the backend API of grant AWS Acccess

### Documentation Updated?

- [x] Yes
- [ ] No

### Under Feature Flag

- [ ] Yes
- [X] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [ ] Yes
- [x] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

File: aws-groups/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# AWS Access
+
+**Note:**: This API can be only called from the discord slash command and which can be only executed by the super users
+
+## **Requests**
+
+[POST /aws/groups](#aws/groups)  - Creates and adds user to the AWS group
```
Comment (samarpan1738) [inline aws-groups/README.md:7]: Wasn't the url `/aws/groups/access`?

File: aws-groups/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+**Note:**: This API can be only called from the discord slash command and which can be only executed by the super users
+
+## **Requests**
+
+[POST /aws/groups](#aws/groups)  - Creates and adds user to the AWS group    
+
+## POST /aws/groups
+
+Returns all users in the system.
```
Comment (samarpan1738) [inline aws-groups/README.md:11]: I think the description is wrong

File: aws-groups/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    ```
+
+- **If user hasn't set their mail address:**
+  - **Code:** 400
+    - **Content:**
+    ```
+       {
+        error: `User email is required to create an AWS user. Please update your email by setting up Profile service, url : ${PROFILE_SVC_GITHUB_URL}` 
+       }
+    ```
```
Comment (samarpan1738) [inline aws-groups/README.md:51]: Can we add these cases under `Error Response` heading?

Comment (vikhyat187) [reply aws-groups/README.md]: updated

File: aws-groups/README.md (hunk 2 above)
Comment (vikhyat187) [reply aws-groups/README.md]: updated

File: aws-groups/README.md (hunk 1 above)
Comment (vikhyat187) [reply aws-groups/README.md:7]: updated, missed this one

Comment (samarpan1738) [reply aws-groups/README.md:7]: It's still showing me `/aws/groups`

**Review by samarpan1738 — APPROVED** [review]

**Pull request merged**